Src/ILGPU/Runtime/OpenCL/CLAPI.xml

```diff
@@ -65,7 +65,7 @@
             ReturnType="IntPtr">
         <Parameter Name="context" Type="IntPtr" />
         <Parameter Name="device" Type="IntPtr" />
-        <Parameter Name="properties" Type="IntPtr" />
+        <Parameter Name="properties" Type="CLCommandQueueProperties" />
```

Comment:
Question: Is this change correct? If so, was it wrong before?

Reply:
It was fine - previously, we always used `IntPtr.Zero`, which is the valid default.